.gitattributes

```diff
@@ -0,0 +1,6 @@
+# Set all files to unix line endings by default
+* text eol=lf
```

Comment:
Sollte das hier nicht `* text=auto eol=lf` und dann bräuchte man png und jpg nicht als binary deklarieren?

Reply:
Ich bin nicht ganz sicher. Ich verstand es so, dass text=auto einfach nur die automatische CRLF-Konvertierung aktiviert (was ja hier eigentlich nicht gewünscht ist).

Quelle: https://www.git-scm.com/docs/gitattributes

"Set to string value "auto"

When text is set to "auto", the path is marked for automatic end-of-line conversion. If Git decides that the content is text, its line endings are converted to LF on checkin. When the file has been committed with CRLF, no conversion is done."

Ich probiere aber gleich mal aus was passiert wenn ich Deinen Vorschlag übernehme.

Reply:
Hab den Build mit Deinen Vorschlag gerade durchlaufen lassen. Sieht gut aus. Commit ist drin, bitte um nochmaliges Review. Thx!

Reply:
Ich hätte die Information direkt von der prettier Seite. Kann sie hier später noch verlinken

Reply:
Ja, gerne. Lerne immer gerne was dazu 👍. Btw. danke für den Merge meines ersten Commits zu einem open source Projekt 🙏

Reply:
https://prettier.io/docs/en/options.html#end-of-line

Reply:
Gerne. Wir freuen uns das du dich engagierst :)